src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -8,11 +8,13 @@ import getProp from 'lodash/get';
 import noop from 'lodash/noop';
 import classNames from 'classnames';
 
+import API from '../../../../api';
```

Comment:
API should not be imported here, it should only be inside ContentSidebar.js which is the main container and the functions to make calls should be passed down.